homeassistant/components/anglian_water/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "domain": "anglian_water",
+  "name": "Anglian Water",
+  "codeowners": ["@pantherale0"],
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/haanglianwater",
+  "iot_class": "cloud_polling",
+  "loggers": ["pyanglianwater", "anglian_water"],
```

Comment:
Why does it have 2 loggers

Reply:
One is for the module that the integration uses, the other is for the integration itself.

The module has a seperate logging namespace.

Reply:
Oh ok, cool. Thanks